rpm/poppler.spec

```diff
@@ -25,8 +25,10 @@ BuildRequires:  pkgconfig(Qt5Widgets)
 BuildRequires:  pkgconfig(Qt5Test)
 BuildRequires:  pkgconfig(libtiff-4)
 BuildRequires:  boost-devel
+BuildRequires:  libjpeg-turbo-devel >= 2.1.0+git3
```

Comment:
Please use `pkgconfig(libjpeg) >= 2.1.0+git3` instead.

Reply:
Tested now, doesn't work:
```
error: Failed build dependencies:
	pkgconfig(libjpeg) >= 2.1.0+git3 is needed by poppler-21.08.0+git1+poppler.22.08.0.20220902172642.01097f4+upstream.eb22dc75-1.aarch64
```
Honestly i have no idea how libjpeg relates to libjpeg-turbo in resolving dependencies, but i'm guessing that is part of the issue.
`BuildRequires:  pkgconfig(libjpeg-turbo) >= 2.1.0+git3`
Does not work either.

Reply:
pkgconfig symbols strip the plus away I guess, in any case libjpeg already exists as buildrequires you should be able to skip that without breaking the build process since obs will rebuild the dependencies first.

Reply:
I'm thinking we should just skip the extra BuildRequires here. Libjpeg can be provided with the original library while the version here is -turbo specific. And if the +git version cannot be used, it's a bit pointless anyhow.

Reply:
I'm fine with anything, just as long as the package builds :)